plugins/linux-capture/pipewire.c

```diff
@@ -552,6 +552,7 @@ static void on_param_changed_cb(void *user_data, uint32_t id,
 	const struct spa_pod *params[3];
 	uint8_t params_buffer[1024];
 	int result;
+	uint32_t buffer_types = 1 << SPA_DATA_MemPtr;
```

Comment:
Nitpick: please move the initial assignment to right before the `if (check_pw_version(&obs_pw->server_version, 0, 3, 24))` line. Like this:

```c
	buffer_types = 1 << SPA_DATA_MemPtr;
	if (check_pw_version(&obs_pw->server_version, 0, 3, 24))
		buffer_types |= 1 << SPA_DATA_DmaBuf;
```